example/100-plant.yaml

```diff
@@ -0,0 +1,21 @@
+# Plant cluster registration manifest through which an external kubernetes cluster will be mapped
+---
+apiVersion: core.gardener.cloud/v1alpha1
+kind: Plant
+metadata:
+  name:  example-plant
+  namespace: garden-dev
+spec:
+  secretRef:
+    name: my-external-cluster
+    namespace: my-namespace
+  monitoring:
```

Comment:
As discussed, let's switch to [that](https://github.com/gardener/gardener/pull/826#issuecomment-473811130) structure.